docs/UserGuide.md

```diff
@@ -111,6 +113,12 @@ Examples:
 * `mark -p John Tan -e Origami workshop -s done` marks the participant `John Tan` in the `Origami workshop` event as present.
 * `mark -p John Tan -e Origami workshop -s undone` marks the participant `John Tan` in the `Origami workshop` event as absent.
 
+### Saving the program: `save`
+
+Save the program
+
+Format: `save`
```

Comment:
I just added this first because I was gonna implement it in my storage class. I have yet to submit a pull request for that

Reply:
Cause I feel we should give the option to the user to save manually in cases where the program actually terminates halfway instead of exiting normally